scripts/bench_native_channels.sh

```
@@ -0,0 +1,84 @@
+[[ -x "$bench_bin" ]] || fail "built binary not executable: $bench_bin"
+
+mkdir -p "$(dirname "$report")"
+{
+	echo "# Native channel request/reply benchmark"
+	echo
+	echo "Generated: $(date -u '+%Y-%m-%dT%H:%M:%SZ')"
+	echo
+	echo "## Environment"
+	echo
+	echo "- surge: $("$surge" version --full | tr '\n' ' ' | sed 's/[[:space:]]*$//')"
+	echo "- fixture: ${fixture#$root/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)